.github/workflows/sync-opencraft-forks-with-upstream.yml

```
@@ -0,0 +1,86 @@
+name: Sync OpenCraft Forks with upstream
+
+# Controls when the workflow will run
+on:
+  # TODO: Uncomment this out when its ready to be merged
+  # schedule:
+  #   - cron: '0 0 * * 1'  # Runs every Monday 00:00 UTC
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  sync:
+    strategy:
+      matrix:
+        include:
+          # The fork repo provided needs to be in the {owner}/{repo} format
+          - fork_repo: open-craft/edx-platform
+            fork_branch: opencraft-release/palm.1
+            upstream_repo: https://github.com/openedx/edx-platform
+            upstream_branch: open-release/palm.master
+
+    runs-on: ubuntu-latest
+    env:
+      # TODO: set this to the OpenCraft Requirements Bot PAT
+      GITHUB_TOKEN: ${{ secrets.GITHUB_BOT_TOKEN }}
```

```suggestion
      GITHUB_TOKEN: ${{ secrets.REQUIREMENTS_BOT_GITHUB_TOKEN }}
```

We already have this.